### DISCUSSION TITLE
Anúncio recorrente de vagas: Falta de qualificação ou Puro marketing?

### DISCUSSION THREAD

**Discussion by n3pu** [frontendbr/forum#979] [Discussão]
Algumas empresas que vejo anunciando vagas, não demoram muito para a nunciar a mesma vaga. Isso eu percebo há uns anos já. Penso eu que isso também é uma forma de dizer "estamos produzindo", mas também, por uma auto avaliação, que poderia ser a falta de qualificação dos candidatos, visto que vários candidatos que conheço sequer são chamados pra entrevista. Isso pode variar a cada região, suponho. Vejo muito isso aqui em Fortaleza. Algo parecido nas demais regiões?

**Comment by kelciocajueiro** [comment]
Eu acredito que é falta de qualificação.

**Comment by viniciusdocanto** [comment]
Olha eu acho que não seja necessariamente isso, hoje em dia todo mundo procura o emprego perfeito, as empresas dão aquela ilusão na entrevista que trabalhar lá você vai ser feliz, depois de um tempo tu vê que não é bem assim e acaba trocando de emprego.

**Comment by NandoSangenetto** [comment]
Não tem como generalizar.

Aqui na empresa, nos últimos 2 meses, anunciamos umas 3 vezes.
Cada um dos anúncios foram em momentos diferentes e por times diferentes. O motivo para tantos anúncios em pouco tempo é que a empresa fechou novos contratos e tomou novas estratégias, surgindo a possibilidade de novas vagas.

Aqui, no caso não é marketing e pensar assim de maneira generalizada acaba sendo injusto com uma empresa que está apenas querendo contratar.

**Comment by felipefialho** [comment]
Acho que cada caso é um caso. 

Tem empresas com grande poder de barganha para contratação que contratam MUITO e continuam abrindo vagas, porque existem demandas lá dentro. 

Por outro lado, tem empresas que sofrem com poucos CVs chegando para vagas e quando chegam, a qualidade muitas vezes não é a necessária (mesmo fazendo busca ativa).
